new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/StdModule.php

```diff
@@ -269,6 +269,23 @@ public function addConfiguration($key, $value, $groupId, $sortOrder, $setFunctio
         xtc_db_query("INSERT INTO `" . TABLE_CONFIGURATION . "` (`configuration_key`, `configuration_value`, `configuration_group_id`, `sort_order`, `set_function`, `use_function`, `date_added`) VALUES ('$key', '$value', '$groupId', '$sortOrder', '$setFunction', '$useFunction', NOW())");
     }
 
+    public function removeConfigurationAll(): bool
+    {
+        $module_name          = $this->getModulePrefix();
+        $remove_configuration = xtc_db_query(
```

Comment:
Nice, but unfortunately not the PRS12 coding style. Please once without spaces before the =